components/common/Header.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+import closeIcon from "@/public/assets/images/landing-page/close-circle.svg";
+import menuIcon from "@/public/assets/images/landing-page/menu.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
+import { Button } from "../ui";
+import LogoBanner from "./LogoBanner";
+
+const Header = () => {
+	const [isMobileNavOpen, setMobileNavOpen] = useState(false);
+
+	const toggleMobileNav = () => {
```

Comment:
We have a useToggle hook in the hooks folder, which was made with this usecase in mind, so you could use that here🤝